srcpkgs/vscode/patches/libgl.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/resources/linux/bin/code.sh b/resources/linux/bin/code.sh
+index 55f50b6f1c14..a083c5f8c10a 100755
+--- a/resources/linux/bin/code.sh
++++ b/resources/linux/bin/code.sh
+@@ -32,5 +32,5 @@ fi
+
+ ELECTRON="$VSCODE_PATH/@@NAME@@"
+ CLI="$VSCODE_PATH/resources/app/out/cli.js"
+-ELECTRON_RUN_AS_NODE=1 "$ELECTRON" "$CLI" "$@"
++LIBGL_ALWAYS_SOFTWARE=1 ELECTRON_RUN_AS_NODE=1 "$ELECTRON" "$CLI" "$@"
```

Comment:
Why is this needed? :thinking:

Reply:
Is it worth that a visual bug (that doesn't seem to get triggered often) degrades performance everywhere?

maybe something like

```sh
if [ -e "${HOME}/.config/code-oss/.libgl_always_software" ]; then
  export LIBGL_ALWAYS_SOFTWARE=1
fi
```

with a notice in the `INSTALL` would be nicer/better?

Reply:
I am working with it since months and didn't notice any performance impact. I think gpu rendering isn't that essential for vscode. But indeed would your solution be better the problem is that if you export LIBGL_ALWAYS_SOFTWARE=1 every other application would be affected too.

Reply:
`export` only affects everything inside the current bash script,

as long as you don't invoke this script via the `source` command in bash everything will be fine.

```sh
echo "Start: ${HELLO}";
bash -c 'export HELLO="world"; echo "Inside: ${HELLO}"';
echo "End: ${HELLO}"
```
results e.g. in:

```
Start:
Inside: world
End:
```

Reply:
Thanks didn't know that. In this case I will update asap and implement your solution. Thanks!

Reply:
Awesome :) thanks!

Reply:
I have to thank you, dude. :)